corehq/apps/reports/tasks.py

```diff
@@ -46,24 +47,25 @@
 logging = get_task_logger(__name__)
 EXPIRE_TIME = ONE_DAY
 
+_calc_props_soft_assert = soft_assert(to='{}@{}'.format('dmore', 'dimagi.com'))
+
 
 @periodic_task(run_every=crontab(hour="22", minute="0", day_of_week="*"), queue='background_queue')
 def update_calculated_properties():
-    success = False
     try:
         _update_calculated_properties()
-        success = True
     except Exception:
+        _calc_props_soft_assert(
+            True,
```

Comment:
I think these should be `False` for it to send.

Reply:
thanks for catching that https://github.com/dimagi/commcare-hq/pull/25591/commits/e9276c0666434eb1c8417508e75a2304f91b7209